src/zcl_abapgit_transport_mass.clas.xml

```diff
@@ -0,0 +1,78 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<abapGit version="v1.0.0" serializer="LCL_OBJECT_CLAS" serializer_version="v1.0.0">
+ <asx:abap xmlns:asx="http://www.sap.com/abapxml" version="1.0">
+  <asx:values>
+   <VSEOCLASS>
+    <CLSNAME>ZCL_ABAPGIT_TRANSPORT_MASS</CLSNAME>
+    <LANGU>E</LANGU>
+    <DESCRIPT>Mass Transport to ZIP</DESCRIPT>
+    <STATE>1</STATE>
+    <CLSCCINCL>X</CLSCCINCL>
+    <FIXPT>X</FIXPT>
+    <UNICODE>X</UNICODE>
+   </VSEOCLASS>
+   <TPOOL>
```

Comment:
text pools does not work in the compiled report, https://raw.githubusercontent.com/abapGit/build/master/zabapgit.abap, so please remove these.

this is bad ABAP programming practice I know

Reply:
Yes, I forgot about this compiled report limitations.. It will be removed in the next commit.